generators/root/templates/deployment/ansible/.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: '2.1'
+orbs:
+  terraform: circleci/terraform@3.0.1
+
+jobs:
+  # TODO: for the deploy job to work you have to:
```

Comment:
I think this should go to the documentation of the project, in the ansible readme: https://github.com/thetribeio/generator-project/blob/main/generators/root/templates/deployment/ansible/ansible/README.md

Reply:
I've moved the note to the README.